pkg/grpc/tracing/tracing.go

```diff
@@ -239,13 +242,34 @@ func GetProcessKprobe(event *MsgGenericKprobeUnix) *tetragon.ProcessKprobe {
 		}
 	}
 
+	var stackTrace []*tetragon.StackTraceEntry
+	for _, addr := range event.StackTrace {
+		if addr != 0 {
+			kernelSymbols, err := ksyms.KernelSymbols()
```

Comment:
What does an `addr` of zero mean? Do we need to issue a warning?

nit/suggestion:

```
if addr == 0 {
    continue
}
/* ... rest of the code goes here */
```

Reply:
It's mostly because the event here is from this struct and we store the stack trace into an array (not a slice) which is already to the size of MAX (127) so we might have a lot of zeros https://github.com/cilium/tetragon/pull/1429/files/086c82354e96092f49008c17a857259d2645c00a..9a5111467b295f2f4e88c7e4f573f0a8c14839fb#diff-09da9c4618bddc46b4f0bfefea390a0d75bf59158a6a0c6d09670e1cfef638a5R420

Reply:
added a comment and use the if addr = 0 { continue } code structure.